xap-enterprise/Dockerfile

```diff
@@ -1,31 +1,31 @@
-ARG JAVA_IMAGE=openjdk
-ARG JAVA_TAG=8-alpine
-FROM ${JAVA_IMAGE}:${JAVA_TAG}
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal
+ARG BASE_TAG=8.4
+FROM ${BASE_IMAGE}:${BASE_TAG}
 ARG GS_PRODUCT=xap
 ARG GS_VERSION=16.0.0
 ARG GS_BUILD_NAME=16.0.0-m21-sun-8
 ARG GS_NAME=gigaspaces-${GS_PRODUCT}-enterprise-${GS_BUILD_NAME}
 ARG GS_URL=https://gigaspaces-releases-eu.s3.amazonaws.com/xap/${GS_VERSION}/${GS_NAME}.zip
 ENV GS_HOME /opt/gigaspaces
 ENV GS_PRODUCT ${GS_PRODUCT}
+ENV KUBERNETES_CLINET_URL=https://resources.gigaspaces.com/k8s-java-client/k8s-java-client.jar
+ENV KUBERNETES_JAVA_LIB_VERSION=5.0.0
 
 WORKDIR ${GS_HOME}
 # Download $GS_URL and unzip to $GS_HOME
-RUN apk --no-cache add openssl wget unzip bash libc6-compat &&\
+USER root
+
+RUN microdnf install java-1.8.0-openjdk-1.8.0.292.b10-1.el8_4.x86_64 jq shadow-utils hostname bind-utils nss wget unzip &&\
```

Comment:
The java version should be an arg so users can build dockerfile with other java version (similar to the java_tag in the prev dockerfile)